src/AutoRest.CSharp/Generation/Writers/LowLevelClientWriter.cs

```diff
@@ -181,67 +179,30 @@ private void WriteSimplifiedConstructor (CodeWriter writer, LowLevelRestClient c
             {
                 writer.WriteXmlDocumentationParameter(parameter.Name, parameter.Description);
             }
+            writer.WriteXmlDocumentationParameter(OptionsVariable, "The options for configuring the client.");
 
-            writer.Append($"public {client.Type.Name:D}(");
-            foreach (Parameter parameter in ctorParams)
-            {
-                writer.WriteParameter(parameter);
-            }
-            writer.RemoveTrailingComma();
-            writer.Append($")");
-
-            // The full ctor params can be in a different order, the options are not necessarily at the end
-            var fullCtorParams = client.GetConstructorParameters(keyCredential ? typeof(AzureKeyCredential) : typeof(TokenCredential), true);
-            writer.Append($": this(");
-            foreach (Parameter parameter in fullCtorParams)
-            {
-                if (parameter.Type.Name != "ProtocolClientOptions")
-                {
-                    writer.Append($"{parameter.Name:D}");
-                }
-                else
-                {
-                    writer.Append($"new {typeof(Azure.Core.ProtocolClientOptions)}()");
-                }
-                writer.AppendRaw(", ");
-            }
-            writer.RemoveTrailingComma();
-            writer.Line($")");
-            using (writer.Scope())
-            {
-            }
-            writer.Line();
-        }
-
-        private void WriteFullConstructor (CodeWriter writer, LowLevelRestClient client, bool keyCredential)
-        {
-            var ctorParams = client.GetConstructorParameters(keyCredential ? typeof(AzureKeyCredential) : typeof(TokenCredential), true);
-
-            writer.WriteXmlDocumentationSummary($"Initializes a new instance of {client.Type.Name}");
-            foreach (Parameter parameter in ctorParams)
-            {
-                writer.WriteXmlDocumentationParameter(parameter.Name, parameter.Description);
-            }
-
+            var clientOptionsName = ClientBase.GetClientPrefix(context.DefaultLibraryName, context);
```

Comment:
Wasn't there a client prefix property on the ClientBase itself?

Reply:
This was cloned from the sister data plane writer:

https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Generation/Writers/DataPlaneClientWriter.cs#L152